src/lib/utils/deepLinking.web.js

```diff
@@ -17,6 +17,11 @@ class DeepLinkingWeb {
 
     return params
   }
+
+  clearQuery() {
+    const urlWithoutQuery = this.link.split('?')[0]
+    window.history.replaceState({}, '', urlWithoutQuery)
```

Comment:
won't it break navigation ? maybe better to call some function from react-navigation ?